frontend/src/features/admin-form/create/builder-and-design/utils/validateNumberInput.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Validates the input given to Chakra's NumberInput component. Pass this to the
+ * onChange parameter for NumberInput. Calls onChange('') if the input is empty,
+ * onChange(number) if the input is a number, and does nothing if the input is
+ * invalid.
+ * @param onChange react Component onChange
+ * @param valStr input value as string
+ * @param valNum input value as number (can be NaN!)
+ * @returns void
+ */
+export const validateNumberInput = (
```

Comment:
nit: seeing this again, should we make this a higher order function so it can be directly instantiated?

```tsx
export const validateNumberInput = (onChange) => (valStr, valNum) => {
  ...
}

// usage
render={({ field: { onChange, ...rest } }) => (
                <NumberInput
                  flex={1}
                  onChange={validateNumberInput(onChange)}
                  {...rest}
                />
              )}

```

Reply:
ok!